packages/framework/src/Component/Collector/ShopsysFrameworkDataCollector.php

```diff
@@ -111,10 +111,13 @@ public function getDocsVersion(): string
      */
     protected function resolveDocsVersion(string $versionString): string
     {
-        if (strpos($versionString, '-dev') !== false) {
-            return 'master';
+        if (!preg_match('~^(\d+)\.(\d+)~', $versionString, $matches)) {
```

Comment:
For current master branch is version string `8.1.0-dev` and in docs.shopsys.com is no `8.1` version (the version is named `latest`).
Unfortunately, I don't have any idea how to do it, as there is no information about branch or so...


And for version parsing could be used `PharIo\Version\Version` like 
```php
// throws exception on invalid version string
$version = new PharIo\Version\Version($versionString)
$version->getMajor()->getValue();
$version->getMinor()->getValue();
```

Reply:
Nice point with `PharIo\Version\Version` 👍 

I also changed resolving versions a bit. See a new condition for more details.
Some examples of resolving:
```
7.3.3-dev -> 7.3
7.3.3     -> 7.3
8.0.1     -> 8.0
8.0.1-dev -> 8.0
8.1.0-dev -> 8.0
8.1.0.    -> 8.1 //I hope we will release also a new doc version
9.0.0-dev -> 9.0
etc.
```